utils/compareImg.py

```diff
@@ -0,0 +1,53 @@
+# pip install pillow
+from PIL import Image
+import numpy as np
+import sys
+
+if len(sys.argv) < 3:
+    sys.exit("Please provide paths to two images.")
+
+# read two images
+img1 = Image.open(sys.argv[1])
+img2 = Image.open(sys.argv[2])
+
+img1_dim = len(np.array(img1).shape)
+img2_dim = len(np.array(img2).shape)
+if img1_dim != img2_dim:
+    print("Please provide images of the same type. Currently, one is a grayscale image and the other is a color image.")
```

Comment:
It might not always be the case when the mismatched images are grayscale and color (one could be `RGB` while the other could be `RGBA` or any other no. of channels?)

Reply:
Only the first sentence remained now. The `RGB` and `RGBA` or other channels is a problem, but im not sure what to write here.